force-app/main/default/classes/Utility.cls

```diff
@@ -0,0 +1,54 @@
+public with sharing class Utility {
+    public static String CONFIG = 'Config';
```

Comment:
You should name it to `CONFIG_NAME`. Purpose of constants is possibility to update then in any time if needed. If you rename it in the future then it will be not so easy to understand what means just `CONFIG` word